x-pack/plugins/security_solution/public/management/components/endpoint_response_actions_list/components/hooks.tsx

```diff
@@ -131,8 +131,11 @@ export const getActionStatus = (status: ResponseActionStatus): string => {
  * running-processes -> processes
  */
 export const getUiCommand = (
-  command: ResponseActions
-): Exclude<ResponseActions, 'unisolate' | 'running-processes'> | 'release' | 'processes' => {
+  command: ResponseActionsApiCommandNames
+):
+  | Exclude<ResponseActionsApiCommandNames, 'unisolate' | 'running-processes'>
+  | 'release'
+  | 'processes' => {
```

Comment:
Now that we have a `ConsoleResponseActionCommands` type, this could be substituted with that.

```suggestion
): ConsoleResponseActionCommands => {
```